api/webhook/post.go

```diff
@@ -191,8 +192,16 @@ func PostWebhook(c *gin.Context) {
 		// send API call to update the webhook
 		_, err = database.FromContext(c).UpdateHook(ctx, h)
 		if err != nil {
-			logrus.Errorf("unable to update webhook %s/%d: %v", r.GetFullName(), h.GetNumber(), err)
+			l.Errorf("unable to update webhook %s/%d: %v", r.GetFullName(), h.GetNumber(), err)
 		}
+
+		l.WithFields(logrus.Fields{
+			"hook":    h.GetNumber(),
+			"hook_id": h.GetID(),
+			"org":     r.GetOrg(),
+			"repo":    r.GetName(),
+			"repo_id": r.GetID(),
+		}).Info("hook updated in database")
```

Comment:
thoughts on getting rid of "in database" 
im feeding off your annotations fixes here https://github.com/go-vela/server/pull/1139
i really liked the removal of the redundant language for "in the configured backend" and "in the database"

Reply:
should be addressed in https://github.com/go-vela/server/pull/1145/commits/53edfd650dbff8ab902ed3490e4fc55bd36ee151